Schemas/MissionHandlers.xsd

```diff
@@ -793,6 +793,10 @@
         "{{{jump 1}}}" - move the agent one block up.
         
         "{{{look 1}}}" - look down by 45 degrees (-ve = up, +ve = down).
+        
+        "{{{attack 1}}}" - destroy the block that has focus.
```

Comment:
We no longer need a parameter in our command syntax - does it make sense to specify a 1 here?